README.adoc

```diff
@@ -0,0 +1,31 @@
+= wildfly-tck-runners
+
+Test runners for executing various TCKs against a WildFly. Each project should have its own README on how to run the
+specific TCK.
+
+== Branches
+
+This project uses branches to target running on specific versions of https://github.com/wildfly/wildfly[WildFly]. For
+example the 31.x branch will be used to run against the 31.x branch of WildFly. 
+
+=== Updating for branching
+
+Once the WildFly branch is crated, a new branch should be created before any new PR's are merged. The branch name
```

Comment:
With around 4 major WildFly releases per year we can expect that over a long period of time we could end up with a lot of branches.  Having said that, this sounds like a good approach.

Reply:
> With around 4 major WildFly releases per year we can expect that over a long period of time we could end up with a lot of branches. Having said that, this sounds like a good approach.

We do this in wildfly/wildfly itself which is why I thought we should do this. After the next micro release we cold always create a tag and remove the branch.

> 
> Just found a typo, `crated` should be `created`.

Nice catch, I'll fix it!

Reply:
Typo fixed, thank you.